build.sbt

```diff
@@ -32,6 +32,24 @@ ThisBuild / wildcardImport := {
   }
 }
 
+// GHA
+
+ThisBuild / githubWorkflowBuild ++= List(
+  WorkflowStep.Sbt(
+    List("${{ matrix.ci }}", "scalafixAll --check"),
+    name = Some("Check Scalafix rules"),
+    cond = Some(s"matrix.scala != '$Scala3'")
+  ),
```

Comment:
Instead of adding this, you should remove these:
https://github.com/typelevel/idna4s/blob/4f609489e745cdb9c2aecbd8ab3c5d13505010b5/build.sbt#L19-L21